### PR TITLE
Install icon in prefix, in proper location, rename .desktop and appdata to proper id name

### PR DESCRIPTION
**Pull request by hfiguiere**

### CODE REVIEW

**Comment by hfiguiere** [comment]
This is a variation on a patch I use for flathub/flathub#819

File: linux/org.hydrogenmusic.hydrogen.appdata.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2015 Aurelien Leblond <blablack@gmail.com> -->
 <component type="desktop">
- <id>hydrogen.desktop</id>
+ <id>org.hydrogenmusic.hydrogen.</id>
```
Comment (bilelmoussaoui) [inline linux/org.hydrogenmusic.hydrogen.appdata.xml]:
```suggestion
 <id>org.hydrogenmusic.hydrogen</id>
```

Comment (hfiguiere) [reply linux/org.hydrogenmusic.hydrogen.appdata.xml]: that's a typo I didn't even notice.

Comment (bilelmoussaoui) [reply linux/org.hydrogenmusic.hydrogen.appdata.xml]: another eye always helps ;)

File: linux/org.hydrogenmusic.Hydrogen.appdata.xml [hunk 2]
```diff
@@ -1,28 +1,43 @@
...
   <screenshot type="default">
-   <image>http://www.hydrogen-music.org/hcms/sites/default/files/pictures/main-window.png</image>
+   <image>http://hydrogen-music.org/images/screenshots/main-window.png</image>
    <caption>The main window showing the application in action</caption>
    <caption xml:lang="fr">La fenêtre principale montrant l'application en action</caption>
   </screenshot>
+  <screenshot>
+    <image>http://hydrogen-music.org/images/screenshots/audio-filebrowser.png</image>
+    <caption>Audio file browser with sample preview</caption>
+    <caption xml:lang="fr">Navigateur de fichiers audio avec previsualisation de l'échantillon</caption>
```
Comment (trebmuh) [inline linux/org.hydrogenmusic.Hydrogen.appdata.xml]: previsualisation -> prévisualisation

File: linux/org.hydrogenmusic.Hydrogen.appdata.xml [hunk 3]
```diff
@@ -1,28 +1,43 @@
...
+  </screenshot>
+  <screenshot>
+    <image>http://hydrogen-music.org/images/screenshots/sample-editor.png</image>
+    <caption>Sample editor</caption>
+    <caption xml:lang="fr">Editeur d'échantillon</caption>
```
Comment (trebmuh) [inline linux/org.hydrogenmusic.Hydrogen.appdata.xml]: Editeur -> Éditeur

**Comment by trebmuh** [comment]
Hi @hfiguiere , I've just made a review with 2 changes needed (French accents), and I'm pinging you here to ask you to resolve the 2 conflicting files please.
Thanks!

**Comment by hfiguiere** [comment]
Rebased. The French spelling with fixed at the same time.

**Comment by hfiguiere** [comment]
This actual revert a changed that rebase caused and shouldn't have. PR is good now.

**Comment by mauser** [comment]
Thanks a lot, merged the pull request!

**Pull request merged**